build_config/lib/src/key_normalization.dart

```diff
@@ -41,11 +41,17 @@ String normalizeTargetKeyDefinition(String targetKey, String packageName) =>
 ///
 /// Example normalizations:
 ///
+///   - "$default" => "$packageName:$packageName"
+///   - "$default:$default": => "$packageName:$packageName"
```

Comment:
I am not sure if we need to actually support this variant, but I guess I don't see a reason why not either?